control-plane/subcommand/gateway-resources/command.go

```diff
@@ -73,6 +78,8 @@ type Command struct {
 	flagDeploymentMaxInstances     int
 	flagDeploymentMinInstances     int
 
+	flagMeshGatewayConfigLocation string
```

Comment:
make this configurable to it's easier to pass in a different filename while testing to test the unmarshal logic